### PR TITLE
fix: redemption rate bug in reconfiguration

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Fixes a bug in the reconfiguration settings where when a user disables the redemption rate, it sets it to 100; instead it now sets it to 0.

I have also removed disabling the save changes button when there is no changes as it is flaky, and may not behave as user expects. As the user can open this modal and then get feedback whether they can actually deploy or not, I think it is okay for now.

This also fixes the ballot redemption rate, by automatically setting it the same as the redemption rate.

## Steps to Repeat Initial Bug

1. Create a project with redemption rate set. For ease of testing, set to 0 < x < 100
1. Open settings and disable redemption rate with switch
1. Deploy
1. See error (redemption rate will be incorrectly set to 100)

It is **expected** that the redemption rate be set to 0.

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4107** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4107" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/4107?utm_source=stack-comment).

File: src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/EditCyclePage.tsx [hunk 1]
```diff
@@ -135,7 +135,8 @@ export function EditCyclePage() {
           <Button
             type="primary"
             onClick={() => setConfirmModalOpen(true)}
-            disabled={!formHasUpdated || Boolean(error)}
+            // Removed the disabled state for now, as it was causing issues.
```
Comment (johnnyd-eth) [inline src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/EditCyclePage.tsx:138]: What was the issue here sir?

**Comment by johnnyd-eth** [comment]
@wraeth-eth The changes I made:
- make initial switch value be disabled if `== 0`, as opposed to `== 100`
- Revert the button disabled state changed, since it was fixed in another PR
- Move reserved rate fields to the bottom of the token diff preview in the "Review and deploy" modal
- Add 'read-only' values to the disabled toggles where relevant (reserved, issuance reduction, redemption)

https://github.com/jbx-protocol/juice-interface/assets/96150256/c5969dbe-4a41-450f-bc85-a49f1ada030f

File: src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/EditCyclePage.tsx (hunk 1 above)
Comment (wraeth-eth) [reply src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/EditCyclePage.tsx]: its just not reliable for this and causes problems.

For instance, if you have advanced closed the button shows up as enabled, then you open advanced and its now disabled.

I think we just remove it

Comment (johnnyd-eth) [reply src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/EditCyclePage/EditCyclePage.tsx]: Ok no worries, I'm down. Done

**Pull request merged**